backoffice/src/resources/PublicUsers/types.tsx

```diff
@@ -75,11 +75,21 @@ export interface CheckHistoryTechnicalDetails {
   registrationDateTime: Date
 }
 
+export enum CheckHistoryStatus {
+  OK = 'ok',
+  KO = 'ko',
+  STARTED = 'started',
+  SUSPISCIOUS = 'suspiscious',
+  PENDING = 'pending',
+  CANCELED = 'canceled',
+  ERROR = 'error',
+}
+
```

Comment:
Est-ce qu'il ne manque pas les valeurs `vide` et `to-do` ?

Reply:
Non, ici in a uniquement les valeurs de status liés au FraudCheck. `vide` et `to-do` n'en font pas parti.